src/libs/terms-of-service-alerts.test.js

```diff
@@ -8,7 +8,11 @@ import * as TosAlerts from 'src/libs/terms-of-service-alerts'
 
 
 jest.mock('src/libs/ajax')
-jest.mock('src/libs/nav')
+
+jest.mock('src/libs/nav', () => ({
```

Comment:
Had to update this mock to prevent `getCurrentUrl` from nav from returning undefined (which causes getCurrentBrand to fail).